### PR TITLE
Elasticsearch GSG review

### PR DESCRIPTION
**Pull request by zcorleissen**
Suggested edits and updates for each document in the Elastic GSG.

### CODE REVIEW

File: source/elastic_faq.rst [hunk 1]
```diff
@@ -1,59 +1,61 @@
...
+ObjectRocket Elasticsearch instances include a cluster with a total of 11 nodes. We automatically configure 4 front-end client nodes, 2 Kibana nodes, 3 master nodes, and 2 data nodes. All nodes are containerized, which provides guaranteed resources (memory, CPU, disk I/O), prevents “noisy neighbor” problems, and eliminates the performance bottlenecks of traditional hardware virtualization.
 
 How much does ObjectRocket Elasticsearch cost per month?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The monthly cost of our ObjectRocket Elasticsearch instances are based on the plan size of the data nodes and the number of data nodes in the cluster.  Client, Kibana, and master nodes are included at no additional charge.
+Prices for ObjectRocket Elasticsearch instances are based on the plan size of the data nodes and the number of data nodes in the cluster.  Client, Kibana, and master nodes are included at no additional charge. Visit `plans and pricing <http://www.objectrocket.com/pricing>`_ and click on **Elasticsearch** for more information.
```
Comment (jtharpla) [inline source/elastic_faq.rst]: We specifically need to identify pricing is a monthly cost, this was requested by the Sales and Billing teams

Comment (zcorleissen) [inline source/elastic_faq.rst]: I'll add "monthly" to the beginning of the line.

File: source/elastic_faq.rst [hunk 2]
```diff
@@ -1,59 +1,61 @@
...
 
-We automatically backup your data every 24 hours using Elasticsearch snapshots.  Contact `support@objectrocket.com <mailto:support@objectrocket.com>`_ to request a restore.
+We automatically back up your data every 24 hours using Elasticsearch snapshots. Contact `support@objectrocket.com <mailto:support@objectrocket.com>`_ to request a restore.
+
+What are the default numbers of shards and replicas per index?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-What are the default number of shards and replicas per index?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+New indexes have 5 primary shards, each with 1 replica shard, for a total of 10 shards for any new index. (ObjectRocket Elasticsearch instances use Elasticsearch default settings of ``number_of_shards = 5`` and ``number_of_replicas = 1``.)
```
Comment (jtharpla) [inline source/elastic_faq.rst]: This one is a very tricky question.  I would prefer we leave the original response to this question, so as to not confuse customers how this works in Elasticsearch.

File: source/elastic_features.rst [hunk 3]
```diff
@@ -3,30 +3,40 @@ Features
 
 Kibana
 ------
-We include an instance of `Kibana <https://www.elastic.co/products/kibana/>`_ automatically configured to connect to your Elasticsearch cluster.  Kibana allows you to build rich visualizations of your Elasticseach data using charts and graphs.
 
-You can view the plain text and SSL URL addresses for Kibana under the **Connect** heading of the Control Panel.  Before you can access Kibana you will need to add Kibana-specific ACLs under the **Security** heading.
+ObjectRocket includes a `Kibana <https://www.elastic.co/products/kibana/>`_ instance automatically configured to connect to your Elasticsearch cluster. Kibana builds rich visualizations of your Elasticseach data with charts and graphs.
+
+Under Instances > Instance Details, the *Connect* heading displays URLs for Kibana connections in plain text and SSL (transport).
+
```
Comment (jtharpla) [inline source/elastic_features.rst]: "Transport" has a very specific meaning in Elasticsearch terminology and does not apply here.

File: source/elastic_getting_started.rst [hunk 4]
```diff
@@ -2,56 +2,99 @@ Getting Started with Elasticsearch
...
-- Click the **Add Instance** button.
-
-- Select a name for your instance. This can be almost anything, as any alpha numeric string is valid.
+#. Enter a name for your instance. 
 
-- Select a Service, Elasticsearch in this case.
+    This can be almost anything, as any alpha numeric string is valid.
 
-- Verify information provided is correct and move on to Step 2
+#. Select Redis for the service. The type defaults to HA Redis.
```
Comment (jtharpla) [inline source/elastic_getting_started.rst]: Copy paste error?

File: source/elastic_utilities.rst [hunk 5]
```diff
@@ -4,22 +4,20 @@ Elasticsearch Utilities
...
-
-Performance metrics shown are based on the articles `Cluster Health <https://www.elastic.co/guide/en/elasticsearch/guide/current/_cluster_health.html>`_ and `Monitoring Individual Nodes <https://www.elastic.co/guide/en/elasticsearch/guide/current/_monitoring_individual_nodes.html>`_ from the Elasticsearch Definitive Guide. Please refer to these articles for further insight as to the significance of each metric.
-
-Find more information about setting up elasticstat: https://github.com/objectrocket/elasticstat
+`Elasticstat <https://github.com/objectrocket/elasticstat>`_ monitors real-time performance of an Elasticsearch cluster from the command line, similar to the Unix utilities `iostat <http://linux.die.net/man/1/iostat>`_ or `vmstat <http://linux.die.net/man/8/vmstat>`_. For more information about Elasticsearch performance metrics, check out `Cluster Health <https://www.elastic.co/guide/en/elasticsearch/guide/current/_cluster_health.html>`_ and `Monitoring Individual Nodes <https://www.elastic.co/guide/en/elasticsearch/guide/current/_monitoring_individual_nodes.html>`_ from the Elasticsearch Definitive Guide.
```
Comment (jtharpla) [inline source/elastic_utilities.rst]: This rewrite implies elasticstat performs the monitoring, it does not.  It is a utility used for performance monitoring, not a monitoring system itself.  Can we please retain the original text here, which accurately describes what elasticstat  is/does

**Comment by jtharpla** [comment]
Please see inline comments.  Also, looks like this needs a rebase.

File: source/elastic_faq.rst (hunk 2 above)
Comment (zcorleissen) [inline source/elastic_faq.rst]: How about:

> ObjectRocket Elasticsearch instances are configured with the standard Elasticsearch default settings of ``number_of_shards = 5`` and ``number_of_replicas = 1``. Any newly created index will have 5 primary shards, each with 1 replica shard (for a total of 10 shards for any new index). 

Does this work?

File: source/elastic_features.rst (hunk 3 above)
Comment (zcorleissen) [inline source/elastic_features.rst]: Noted. I'll remove it.

File: source/elastic_getting_started.rst (hunk 4 above)
Comment (zcorleissen) [inline source/elastic_getting_started.rst]: Yes, good catch.

File: source/elastic_utilities.rst (hunk 5 above)
Comment (zcorleissen) [inline source/elastic_utilities.rst]: How about:

> Elasticstat is a utility for real-time performance monitoring of an Elasticsearch cluster from the command line, similar to the Unix utilities iostat or vmstat. The optional parameter DELAYINTERVAL controls update frequency by specifying a delay in seconds after each update.

**Comment by zcorleissen** [comment]
Thanks for the comments. It does need a rebase, but I don't have write access to perform it.

**Pull request closed without merging**